### PR TITLE
Add links to content

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
Make all the links on the landing pages work correctly.

## Why
Improve the navigation of the pages for development purposes.

## Visual changes
Includes some text changes to links, but nothing else.

Trello card: https://trello.com/c/WG11pcLc

### CODE REVIEW

File: lib/data/landing_page_content_items/homepage.yaml [hunk 1]
```diff
@@ -53,8 +53,8 @@ blocks:
   featured_content:
     blocks:
       - type: heading
-        content: Our tasks
-        path: /landing-page/tasks
+        content: Our progress
+        path: /landing-page/goals
```
Comment (AshGDS) [inline lib/data/landing_page_content_items/homepage.yaml:57]: I think this is the wrong page, the design shows it linking to the tasks page.

**Review by leenagupte — APPROVED** [review]

**Review by AshGDS — APPROVED** [review]
Looks good, though I just did the `featured` block on the homepage in https://github.com/alphagov/frontend/pull/4323/ and looks like we've disagreed on what link should go there, so I have left a comment.

File: lib/data/landing_page_content_items/homepage.yaml (hunk 1 above)
Comment (andysellick) [reply lib/data/landing_page_content_items/homepage.yaml:57]: Ah okay thanks will check.

Comment (andysellick) [reply lib/data/landing_page_content_items/homepage.yaml]: Changed it back, sorry, think that was a rebase error.

**Review by AshGDS — APPROVED** [review]

**Pull request merged**